src/format-files.test.js

```diff
@@ -11,6 +11,7 @@ jest.mock('fs')
 
 beforeEach(() => {
   process.stdout.write = jest.fn()
+  process.exitCode = 0
```

Comment:
Should we put this in an afterEach to avoid the process running this test from having an exit code of non-0?